src/screens/HomeScreen.tsx

```diff
@@ -124,6 +124,11 @@ export class HomeScreen extends UIScreen<HomeScreenProps, HomeScreenState> {
             );
         }
 
+        // don't render the message <Text> if the message is empty
+        const messageComponent = this.state.messageText
+            ? <Text style={styles.message}>{this.state.messageText}</Text>
+            : null;
```

Comment:
Will this cause the clock to jump downwards when an alarm message is re-rendered? Since the component isn't being rendered.

Reply:
that's a good question, to which the answer is probably yes

Reply:
changing